app/models/person.rb

```diff
@@ -136,4 +136,9 @@ def ensure_primary_person
       family.update!(primary_person_id: id)
     end
   end
+
+  def child?
+    if person.type == 'Child'
+    end
+  end
```

Comment:
- I think instances of Child should be of the Child class, so we can define the behaviour using OOP, instead of if statements. But maybe the importer is not instantiating them as Child objects?
- I don't think Cru is going to import childcare data for non-children, so filtering that in the importer might not be necessary.